src/main/java/lotto/Entity/WinLotto.java

```diff
@@ -0,0 +1,32 @@
+package lotto.Entity;
+
+import java.util.ArrayList;
+import java.util.List;
+import lotto.Exception.CheckWinningLotto;
+
+public class WinLotto {
+    // 당첨 로또
+    public final List<Integer> winningLotto;
+
+    public WinLotto() {
+        this.winningLotto = new ArrayList<>();
+    }
+
+
+    public List<Integer> drawNumber(String[] winningNum) {
+        CheckWinningLotto.checkListRange(winningNum);
+        for (String number : winningNum) {
+            this.winningLotto.add(
+                CheckWinningLotto.checkNumberRange(Integer.parseInt(number.trim())));
```

Comment:
[nit] 최적화가 되어서 큰 차이는 없지만 parsInt는 primitive type인 int를 반환하고 valueOf는 Integer를 반환합니다.
```suggestion
                CheckWinningLotto.checkNumberRange(Integer.valueOf(number.trim())));
```